### PR TITLE
1.4.12 changelog pull dagster-ext line

### PR DESCRIPTION
**Pull request by alangenfeld**

### CODE REVIEW

**Comment by alangenfeld** [comment]
Current dependencies on/for this PR:
* master
  * **PR #16381** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/16381" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/dagster-io/dagster/16381?utm_source=stack-comment).

File: CHANGES.md [hunk 1]
```diff
@@ -24,16 +25,15 @@
...
-    ```python
-    @asset_check(asset=my_asset)
-    def my_check(my_asset) -> AssetCheckResult:
-        ...
-    ```
+  ```python
+  @asset_check(asset=my_asset)
+  def my_check(my_asset) -> AssetCheckResult:
+      ...
+  ```
```
Comment (schrockn) [inline CHANGES.md:36]: ?

File: CHANGES.md [hunk 2]
```diff
@@ -16,6 +16,7 @@
 - [ui] Fixed an issue where the Raw Compute Logs dropdown on the Run page sometimes didn’t show the current step name or properly account for retried steps.
 
 ### Community Contributions
+
```
Comment (schrockn) [inline CHANGES.md:19]: ? rm?

**Review by schrockn — APPROVED** [review]
Accepted, but either fix extraneous noise or justify plz.

**Comment by alangenfeld** [comment]
other fixes are from markdown auto formatter, rest of changelog complies. Landing this to unblock

**Pull request merged**